openapi.yaml

```diff
@@ -313,10 +313,13 @@ paths:
             type: string
           example: MyAutomationJobName
         - name: from
+          required: true
           in: query
           description: |
             Retrieve runs that started at a time after this date and time.
             If you do not specify this parameter, then a default of 'now' minus 24 hours is assumed. 
```

Comment:
> If you do not specify this parameter, then a default of 'now' minus 24 hours is assumed.

I don't believe that's true when the runname is set ? 

We insist on the value so we don't have to do this weird defaulting to 24 hours I thought.